src/main/kotlin/cacophony/controlflow/functions/FunctionHandler.kt

```diff
@@ -16,7 +16,7 @@ interface FunctionHandler {
     fun registerVariableAllocation(variable: Variable, allocation: VariableAllocation)
 
     // Returns static link to parent
-    fun getStaticLink(): Variable.AuxVariable.StaticLinkVariable
+    fun getStaticLink(): Variable
```

Comment:
Maybe: 'TODO: change it to PrimitiveVariable' instead of generic Variable as return type here?